docs/docs/usage/scaladoc/settings.md

```diff
@@ -40,17 +40,17 @@ Revision (branch or ref) used to build project project. Useful with sourcelinks
 Source links provide a mapping between file in documentation and code repository.
 
 Accepted formats:
-<\sub-path>=<\source-link>
-<\source-link>
+\<sub-path>=\<source-link>
+\<source-link>
 
-where <\source-link> is one of following:
+where <source-link> is one of following:
```

Comment:
backslash?

Reply:
The line 46 does not render correctly.

Reply:
So does line 55, my bad. Thanks for staying alert :D 
Fixed